content/applications/finance/fiscal_localizations/new_zealand.rst

```diff
@@ -2,13 +2,364 @@
+:guilabel:`Send Money` option defines the bank account as trusted, which is essential for EFT files
+and *must be enabled* if it is to be used.
+
+.. _localizations/new_zealand/eft-contacts:
+
+Contacts' bank accounts
+***********************
+
+To add banking information for a contact, navigate to
+:menuselection:`Accounting --> Customers --> Customers`,
+:menuselection:`Accounting --> Vendors --> Vendors`, or access the contact directly through the
+**Contacts** app. Select the relevant contact, then open the :guilabel:`Accounting` tab. Under the
+:guilabel:`Bank Accounts` section, click :guilabel:`Add a line` to enter the required details.
+
+- :guilabel:`Account Number`
+- :guilabel:`Bank`
+- :guilabel:`Account Holder` (it will automatically be selected for that contact)
+- :guilabel:`Send Money` must be **enabled**.
+
+.. _localizations/new_zealand/eft-generate:
+
+Generate an EFT file
+~~~~~~~~~~~~~~~~~~~~
+
+Go to :menuselection:`Accounting --> Customers --> Invoices`
+or :menuselection:`Accounting --> Vendors --> Bills`. In the list view, select the invoices/bills to
+be paid from the list and click :guilabel:`Pay`. In the :guilabel:`Payment Method` field, select
+:guilabel:`New Zealand EFT` and click :guilabel:`Create Payment`.
+
+.. note::
+   The :guilabel:`Group Payment` checkbox is optional. This option appears only if there are
+   multiple invoices or bills from the same contact.
+
+In the Payments window, the required EFT information for each payment, such as
+:guilabel:`Particulars` and :guilabel:`Analysis Code`, can be entered as needed.
+
+.. note::
+   The same payment information can be found under :menuselection:`Accounting --> Customers -->
+   Payments` or :menuselection:`Accounting --> Vendors --> Payments`.
+
+Then, return to the **Payments** list view, select the payments needing to batch, and click
+:guilabel:`Create Batch`.
+
+In the **Batch Payment** view, fill in the following fields:
+
+- :guilabel:`EFT File Format`
+- :guilabel:`Payment Reference`
+- :guilabel:`Payment Particulars`
+
+.. image:: new_zealand/batch_payment_view.png
+   :alt: Batch Payement.
+
+Then, click :guilabel:`Validate`. Odoo will generate the EFT file in the chatter. Click on the file
+to preview or download it directly to the computer.
+
+.. important::
+   Each bank has its specific format requirements for EFT Batch Payments. Make sure to choose the
+   correct EFT file format. Some banks may also require the completion of additional fields, such as
+   :guilabel:`Direct Debit Information` and :guilabel:`Dishonour Account`.
+
+.. seealso::
+   :doc:`Batch Payment <../../../applications/finance/accounting/payments/batch>`
+
+.. _localizations/new_zealand/XXXXXX:
+
+Industry-specific features
+==========================
+
+.. _localizations/new_zealand/starshipit:
+
+Starshipit Shipping
+-------------------
+
+:doc:`Starshipit <../../../applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/starshipit_shipping>`
```

Comment:
I would put the hyperlink to the documentation page in the seealso section. Maybe include the link to Starshipit itself here?